packages/charting/src/Styling.ts

```diff
@@ -0,0 +1 @@
+export * from 'office-ui-fabric-react/lib/Styling';
```

Comment:
This and the Utilities.ts don't seem to be needed - perhaps we can just remove these?

Reply:
They're used in VerticalBarChart.types.ts